CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 2.3.1 (2025-09-15)
+--------------------------
+Bring back the jar version (#162)
```

Comment:
The commit is `Support building a fat jar`, usually we don't change them in the CHANGELOG

Reply:
Oops! I wanted something more user-oriented in the changelog... But I am sure nobody will notice :P